### DISCUSSION TITLE
Searching File field

### DISCUSSION THREAD

**Discussion by Calorion** [RafaelGB/obsidian-db-folder#608] [Ideas]
How do I search only in the File field?

**Comment by RafaelGB** [comment]
There is only a global field. particular search are not implemented yet

**Reply by Calorion** [reply]
But I can make a Filter with other fields. Why doesn't File show up there?

**Reply by RafaelGB** [reply]
File is not an actual field. It's metadata information

**Reply by Calorion** [reply]
That's…frustrating. Well, consider this a request for a way to search or filter on the File field, or at least on the filenames of the relevant files.

**Reply by RafaelGB** [reply]
you can search but not filter

**Reply by Calorion** [reply]
I meant "limit searches to the File field"

**Reply by RafaelGB** [reply]
https://user-images.githubusercontent.com/11924043/202793459-c1d4ceb9-fcf1-47b5-a5cf-526c1dbe9566.MOV


I am working on custom filters (react-table already support them)

currently I am playing with texts and numbers. Other kind of objects like Links (Files) could be an option

**Reply by RafaelGB** [reply]
https://user-images.githubusercontent.com/11924043/202867621-ef967d18-324d-4c89-a594-2d9612e3252a.mov

=)

**Comment by RafaelGB** [comment]
Commit 76f5d331

**Comment by sm-alejandro** [comment]
Related to this, would it be possible to access the search menu or the global search dialog with a keyboard shortcut/command palette command that then can be mapped to a shortcut? It's not a very important feature but would be nice to have. (for completeness). Thank you 😄

**Reply by RafaelGB** [reply]
https://github.com/RafaelGB/obsidian-db-folder/issues/577

Keyboard actions will be the idea with this issue on progress :)

**Reply by sm-alejandro** [reply]
Ah okay, perfect. Thanks 👍

**Reply by sm-alejandro** [reply]
I've seen that you've implemented it. Ty, link here for anyone who sees this later https://github.com/RafaelGB/obsidian-db-folder/issues/577#issuecomment-1327602588

**Reply by RafaelGB** [reply]
yeah, there are a lot of new features. The next version could be v3.0.0 of the plugin :)
